docs/web_reference.rst

```diff
@@ -2181,8 +2189,8 @@ In general the result may be any object derived from
 
 .. class:: UrlMappingMatchInfo
 
-   Inherited from :class:`dict` and :class:`AbstractMatchInfo`. Dict
-   items are filled by matching info and is :term:`resource`\-specific.
+   Inherited from :class:`collections.ChainMap` and :class:`AbstractMatchInfo`.
```

Comment:
The doc is not relevant anymore, `Mapping` should replace `ChainMap`.
Also `.variable_maps` or `.maps` attribute should be documented.